pkgs/tools/admin/pulumi-packages/base.nix

```diff
@@ -91,14 +90,19 @@ let
     inherit owner repo rev hash fetchSubmodules;
   };
 
-  pulumi-gen = mkBasePackage rec {
+  pulumi-gen = mkBasePackage buildPackages.buildGoModule {
     inherit src version vendorHash extraLdflags;
 
     cmd = cmdGen;
     pname = cmdGen;
+
+    subPackages = [
+      "cmd/${cmdGen}"
+      "cmd/${cmdRes}/generate.go"
```

Comment:
This breaks a few plugins that don't have a `generate.go` (`pulumi-aws-native`, `pulumi-azure-native` and `pulumi-command`)

Reply:
Oh, didn’t notice this, sorry. I’ll fix this after upstream another (and last, for now) PR for pulumiPackages.

Reply:
Should be fixed.